src/app/Plans/components/Wizard/PlanWizard.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import WizardStepContainer from './WizardStepContainer';
+import { Wizard } from '@patternfly/react-core';
+import Review from './Review';
+import GeneralForm from './GeneralForm';
+import { Provider } from '@app/Providers/types';
+
+interface IPlanWizardProps {
+  isOpen: boolean;
+  onClose: () => void;
+  sourceProviders: Provider[];
+  targetProviders: Provider[];
+}
+
+const PlanWizard: React.FunctionComponent<IPlanWizardProps> = ({
+  isOpen,
+  onClose,
+  sourceProviders,
+  targetProviders,
+}: IPlanWizardProps) => {
+  enum stepId {
+    General = 1,
```

Comment:
Not a request, just a question: what's the deal with the `= 1` on General but none on the other steps? Looking in the debugger it seems that sets all of these to numeric values, which is pretty cool, I just wasn't familiar with this pattern.

Reply:
@mturley, I actually copied that one from mig-ui, interestingly this pattern is similar in Golang.

Reply:
Cool! Good to know.